### PR TITLE
empty list parameter info

### PR DESCRIPTION
**Pull request by jclark-vmware**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Tests have been added that prove the fix is effective or that the feature works.
- [x] New and existing tests pass locally with the changes.
- [x] Code follows the style guidelines of this project (PEP8, clean code, linter).
- [x] A self-review of the code has been done.

## What is the ticket or issue number?

https://jira.carbonblack.local/browse/CBAPI-3009

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by abowersox-cb — APPROVED** [review]
The flake8 failure isn't anything you did; somehow, it slipped through into develop and will have to get fixed elsewhere.

**Comment by abowersox-cb** [comment]
I fixed the flake8 issue in my own PR.

**Review by avanbrunt-cb — APPROVED** [review]

File: docs/alerts.rst [hunk 1]
```diff
@@ -48,6 +48,15 @@ for more complex searches. The example below will search with a solr query searc
...
     aab3c640-<truncated> WINDOWS WINDOWS-TEST THREAT
 
+.. tip::
+    When filtering by fields that take a list parameter, an empty list will be treated as a wildcard and match everything.
+
+Ex: Returns all types
+
+.. code-block:: python
+
+    >>> alerts = (cb.select(BaseAlert).set_types([]))
```
Comment (avanbrunt-cb) [inline docs/alerts.rst:58]: Can make this `list()` instead of using the parantheses shortcut

**Review by emitreva-cb — APPROVED** [review]

File: docs/alerts.rst (hunk 1 above)
Comment (jclark-vmware) [reply docs/alerts.rst]: done

**Pull request merged**